app/src/test/java/org/mozilla/focus/searchsuggestions/SearchSuggestionsViewModelTest.kt

```diff
@@ -0,0 +1,33 @@
+package org.mozilla.focus.searchsuggestions
+
+import android.arch.lifecycle.Lifecycle
+import android.arch.lifecycle.LifecycleOwner
+import android.arch.lifecycle.LifecycleRegistry
+import org.junit.Test
+
+import org.junit.Assert.*
```

Comment:
\[detekt\] INFO: \[style/WildcardImport\] Wildcard imports should be replaced with imports using fully qualified class names\. Wildcard imports can lead to naming conflicts\. A library update can introduce naming clashes with your classes which results in compilation errors\.